main.go

```diff
@@ -179,13 +179,17 @@ func init() {
 			fmt.Println(config.Config.String())
 		},
 	})
-	rootCmd.AddCommand(&cobra.Command{
-		Use:     "exec",
-		Short:   "Execute a command serverlessly",
+	execCmd := &cobra.Command{
 		GroupID: groupDev,
-		Hidden:  true,
+		Short:   "Execute a command serverlessly",
+		Use:     "exec [flags] -- [<command>]...",
```

Comment:
You shouldn't need to add the `[flags]` manually here. Cobra will do it for you (I think)

Reply:
This `Use` overrides the default that Cobra will do (where it includes flags), but I wanted to include the `[command]` section, which I didn't find an in-built way in Cobra to do. 

Do you know one? Or I can leave it like this?